websocket.go

```diff
@@ -73,14 +73,46 @@ func (p *WebSocketProxy) HandleWebSocket(w http.ResponseWriter, r *http.Request)
 	}
 	defer conn.CloseNow()
 
-	authHeader := r.Header.Get("Authorization")
-	apiKey := ""
-	if strings.HasPrefix(authHeader, "Bearer ") {
-		apiKey = strings.TrimPrefix(authHeader, "Bearer ")
-		log.Printf("Extracted API key (length): %d", len(apiKey))
-	}
-
 	if p.config.CACert != nil {
```

Comment:
Perhaps protect this also with the special env variable like with http

Reply:
The verification of the `DANGEROUS_NO_CA_CERT` is done in `LoadConfig()` so it covers both http and ws. The only way for the ca cert to be nil is if that var is set to `YES`